ui/admin/app/styles/app.scss

```diff
@@ -623,6 +623,24 @@
   }
 }
 
+.target-sidebar-aliases {
```

Comment:
I don't think this is related to this PR (but I don't know if it existed since our last release) but I noticed the migration button for me is...wonky

https://github.com/hashicorp/boundary-ui/assets/5783847/0766fdc6-5be0-4f50-aca0-9803052e6937

Reply:
I'm guessing this has always been there due to the container wrapper that we have inside `F.control` . I moved the button within the container wrapper, and it seemed to fix the issue. Can you check if you are still seeing this?